UI/Pages/TuitionPartner.cshtml

```diff
@@ -5,9 +5,9 @@
 }
 <a asp-page="@nameof(Index)" class="govuk-link" data-testid="home-link">Home</a><br/>
 
-<govuk-back-link href="/search-results?@Model.AllSearchData.ToQueryString()">
+<a href="/search-results?@Model.AllSearchData.ToQueryString()" class="govuk-back-link">
```

Comment:
I'm curious, what was going wrong when using the tag helper?

Reply:
Frustratingly it was HTML encoding rather than URL encoding the href so & became &amp; which broke the back links again.

I had a brief look at the library's source and couldn't see where that was happening. I'll raise it as a post private beta task to look into it and perhaps raise a PR for that library